mod_app/admin/film_admin.py

```diff
@@ -103,32 +126,31 @@ def safe_comments(self, obj):
                 ),
             },
         ),
-        (
-            "Additional Information (non filmic)",
-            {
-                "classes": ("grp-collapse grp-open",),
-                "fields": ("additional_links",),
-            },
-        ),
         (
             "Printed Materials",
-            {
-                "classes": ("grp-collapse grp-open",),
-                "fields": (
-                    ("scripts", "press_books"),
-                    ("programmes", "pub_mat"),
-                ),
-            },
+            {"fields": []},
         ),
+        (None, {"classes": ("placeholder scripts-group",), "fields": ()}),
+        (None, {"classes": ("placeholder pressbooks-group",), "fields": ()}),
+        (None, {"classes": ("placeholder programmes-group",), "fields": ()}),
+        (None, {"classes": ("placeholder publicitys-group",), "fields": ()}),
         (
```

Comment:
Is the idea here that the empty grey printed materials section acts as a sort of section header/divider? I found it a little confusing when trying to understand what I was looking at, but I presume you'd have to get into writing custom templates or something to get exactly what you're after?

Reply:
Yes, the grey printed materials is acting as a header because you can't typically just choose where you want to put your inlines, I think custom templates would let me do that but I don't really want to go down that route unless absolutely necessary; Grappelli provides the next best thing

Reply:
Yeah, fair enough.